### PR TITLE
Ppr/Mhr Search UXA

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity###

*Description of changes:*
* Misc uxa and code clean up

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Looks good,

File: ppr-ui/src/components/search/SearchBarList.vue [hunk 1]
```diff
@@ -138,12 +138,15 @@ export default defineComponent({
         }
       }),
       isSingleSearchOption: computed((): boolean => {
-        return (hasPprEnabled.value && !hasMhrEnabled.value) || (!hasPprEnabled.value && hasMhrEnabled.value)
+        return ((hasPprEnabled.value && !hasMhrEnabled.value) || (!hasPprEnabled.value && hasMhrEnabled.value)) &&
```
Comment (dimak1) [inline ppr-ui/src/components/search/SearchBarList.vue:141]: Looks like we'll be using these combo flags a lot, maybe we can combine them in a new PR.

**Review by dimak1 — APPROVED** [review]

**Comment by dimak1** [comment]
Is this the fix for the bug or simply refactor? @cameron-eyds

File: ppr-ui/src/components/search/SearchBarList.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/search/SearchBarList.vue:141]: Definitely!

**Comment by cameron-eyds** [comment]
> Is this the fix for the bug or simply refactor? @cameron-eyds

A bit of both!
UX review called for some updated behaviour on when the MHR message shows. 
The refactor also fixes the condition for when Staff see the messaging. With the previous condition i put in, it was displaying when the staff accounts had the products as well, which wasn't considered originally.

**Review by ChasBoucher — APPROVED** [review]
Looks good.

**Pull request merged**